app/models/user.rb

```diff
@@ -5,8 +5,21 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
 
   validates :name, presence: true, length: { maximum: 20 }
+  has_many :friendships, dependent: :destroy
 
```

Comment:
You are missing the inverse friendships so that you can find it no matter who sent it. See [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone5/friend.png).